drivers/gles3/rasterizer_scene_gles3.h

```diff
@@ -328,6 +330,7 @@ class RasterizerSceneGLES3 : public RasterizerScene {
 		float atlas_clamp[4];
 		float local_matrix[16]; //up to here for spot and omni, rest is for directional
 		//notes: for ambientblend, use distance to edge to blend between already existing global environment
+		uint8_t padding[10];
```

Comment:
> padding is not needed here

I thought that would screw up our offset calculation in the array. Are they tightly packed when defined as array in the uniform block object?